pkg/commands/apply/apply.go

```diff
@@ -149,20 +151,24 @@ func (c *ApplyCommand) Run(ctx context.Context, args []string) error {
 	}
 
 	parsedArgs := f.Args()
-	if len(parsedArgs) != 1 {
+	if len(parsedArgs) > 0 {
 		return flag.ErrHelp
 	}
 
-	dirAbs, err := util.PathEvalAbs(parsedArgs[0])
+	cwd, err := c.WorkingDir()
 	if err != nil {
-		return fmt.Errorf("failed to absolute path for directory: %w", err)
+		return fmt.Errorf("failed to get current working directory: %w", err)
 	}
-	c.directory = dirAbs
 
-	cwd, err := c.WorkingDir()
+	if c.FlagDir == "" {
```

Comment:
Should this be `c.CommonFlags.FlagDir`? (throughout)

Reply:
Due to the way Go handles Struct embedding, `c.FlagDir` and `c.CommonFlags.FlagDir` are equivalent names for the same field. I can change it if you find the other way more readable.

Reply:
Per offline discussion, we can leave this.